apps/web/lib/d/[link]/[slug]/getServerSideProps.tsx

```diff
@@ -25,41 +25,48 @@ async function getUserPageProps(context: GetServerSidePropsContext) {
   const { currentOrgDomain, isValidOrgDomain } = orgDomainConfig(context.req);
   const org = isValidOrgDomain ? currentOrgDomain : null;
 
-  const hashedLink = await prisma.hashedLink.findUnique({
-    where: {
-      link,
-    },
-    select: {
-      eventTypeId: true,
-      eventType: {
-        select: {
-          users: {
-            select: {
-              username: true,
-              profiles: {
-                select: {
-                  id: true,
-                  organizationId: true,
-                  username: true,
-                },
+  const hashedLinkSelect = {
+    id: true,
+    link: true,
+    eventTypeId: true,
+    expiresAt: true,
+    maxUsageCount: true,
+    usageCount: true,
+    eventType: {
+      select: {
+        users: {
+          select: {
+            username: true,
+            profiles: {
+              select: {
+                id: true,
+                organizationId: true,
+                username: true,
               },
             },
           },
-          team: {
-            select: {
-              id: true,
-              slug: true,
-              hideBranding: true,
-              parent: {
-                select: {
-                  hideBranding: true,
-                },
+        },
+        team: {
+          select: {
+            id: true,
+            slug: true,
+            hideBranding: true,
+            parent: {
+              select: {
+                hideBranding: true,
               },
             },
           },
         },
       },
     },
+  } as const;
```

Comment:
Here instead of `as const` you can use `satisfies Prisma.HashedLinkSelect` for better typing.